framework/tlvf/AutoGenerated/include/tlvf/CmduMessageTx.h

```diff
@@ -26,7 +26,7 @@ class CmduMessageTx : public CmduMessage {
 
 public:
```

Comment:
Please commit the AutoGenerated files in different commit

Reply:
In cases where there are not that many auto generated files, I find it easier to keep one commit for both. I think this is the case here - but if you and @arnout think otherwise, I'll split it.

Reply:
As you know, my POV is that AutoGenerated should not be included at all :-)

The only reason to make it a separate commit is to make review easier - but it doesn't make review that much easier, so I'm fine with it either way.